mmpose/models/detectors/top_down.py

```diff
@@ -179,7 +179,10 @@ def forward_test(self, img, img_metas, **kwargs):
 
         c = img_metas['center'].reshape(1, -1)
         s = img_metas['scale'].reshape(1, -1)
-        score = np.array(img_metas['bbox_score']).reshape(-1)
+
+        score = 1.0
```

Comment:
since it is only used at the right hand side https://github.com/open-mmlab/mmpose/blob/hd/trb/mmpose/models/detectors/top_down.py#L204, this is okay actually

Reply:
Ok.